pennylane/drawer/draw.py

```diff
@@ -449,8 +452,12 @@ def circuit2(x, y):
             box1 = plt.Rectangle((-0.5, -0.5), width=3.0, height=4.0, **options)
             ax.add_patch(box1)
 
-            ax.annotate("CSWAP", xy=(3, 2.5), xycoords='data', xytext=(3.8,1.5), textcoords='data',
+            ax.annotate("CSWAP", xy=(5, 2.5), xycoords='data', xytext=(5.8,1.5), textcoords='data',
                         arrowprops={'facecolor': 'black'}, fontsize=14)
+
+            ax.annotate("classical control flow", xy=(3.5, 4.2), xycoords='data', xytext=(0.8,4.2),
+                        textcoords='data', arrowprops={'facecolor': 'blue'}, fontsize=14,
+                        va="center")
             fig.show()
 
         .. figure:: ../../_static/draw_mpl/postprocessing.png

```

Comment:
This changes the figure:

![Figure_1](https://github.com/PennyLaneAI/pennylane/assets/34751083/faee38d0-c130-407c-a85c-cecd11382454)

Reply:
Ah yes, for some reason the new file was not included in the commit.